less/styles.less

```diff
@@ -211,6 +211,26 @@ html.no-js .sidebar {
     padding: @grid-gutter-width / 2;
   }
 
+  .sidebar-menu li img{
+    float: none;
+    width: 5.4rem;
+    opacity: 0.6;
+    margin: 0 1.5rem 0 0;
+    @media (min-width: @screen-md-min) {
+      float: left;
+    }
+  }
+
+  .sidebar-menu li:nth-child(3) img {
```

Comment:
I wanted to avoid using superflous classes, but what you say makes sense. I will defer to target with a class.